docs/types/value-types.rst

```diff
@@ -65,10 +65,10 @@ This means that, for example ``~int256(0) == int256(-1)``.
 Shifts
 ^^^^^^
 
-The result of a shift operation has the type of the left operand, truncating the result to match the type.
+This means that contrary to checked arithmetic, if the result of a bit shift operation will not fit its type, the transaction will *not* revert.
 Right operand must be unsigned type. Trying to shift by signed type will produce a compilation error.
 
-- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y``.
+- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y`` run in unchecked mode.
```

Comment:
We do have this documented in the section about `checked` and `unchecked`: https://docs.soliditylang.org/en/v0.8.4/control-structures.html?highlight=bitwise%20operators%20do%20not#checked-or-unchecked-arithmetic


> Bitwise operators do not perform overflow or underflow checks. This is particularly visible when using bitwise shifts (<<, >>, <<=, >>=) in place of integer division and multiplication by a power of 2. For example type(uint256).max << 3 does not revert even though type(uint256).max * 8 would.

----

Anyway, good to have it mentioned here as well.